proto/rill/admin/v1/api.proto

```diff
@@ -1287,12 +1313,25 @@ message ListServicesRequest {
 }
 
 message ListServicesResponse {
-  repeated Service services = 1;
+  repeated OrganizationMemberService services = 1;
+}
+
+message ListProjectMemberServicesRequest {
+  string organization_name = 1 [(validate.rules).string.min_len = 1];
+  string project_name = 2 [(validate.rules).string.min_len = 1];
+}
+
+message ListProjectMemberServicesResponse {
+  repeated ProjectMemberService services = 1;
 }
 
 message CreateServiceRequest {
   string name = 1 [(validate.rules).string.min_len = 1];
   string organization_name = 2 [(validate.rules).string.min_len = 1];
+  string org_role_name = 3; // Optional org role to assign
+  string project_name = 4; // Optional project to assign role to
+  string project_role_name = 5; // Optional project role to assign
+  map<string, string> attributes = 6;
```

Comment:
Would about non-string types, like `admin` (bool) or `groups` (array)? Consider using either `google.protobuf.Struct` passing it as JSON, e.g. `string attributes_json`.

If you use `google.protobuf.Struct`, I think you can add `optional` in front – maybe that solves the issue about ability to clear attributes?